### PR TITLE
Add script for easier integration testing

### LINKED ISSUE #440: Write a script for easier integration testing locally
Blocked on #439. Currently there are a few steps and environment variables you need to set for the integration tests to work nicely, we should be able to automate most of that through a simple script.

### PR DESCRIPTION
**Pull request by thieman**
@jsingle Fixes #440. Need to write some docs to go with it still.

### CODE REVIEW

File: tests/run_integration_tests.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Check that the script is being run correctly and can find the Dusty executable
+
+if [ -z ${VIRTUAL_ENV+x} ]; then
```
Comment (jsingle) [inline tests/run_integration_tests.sh]: crazy

**Comment by jsingle** [comment]
Pretty cool if this works.  You gonna change the jenkins job to use this?

**Comment by thieman** [comment]
Eh if it ain't broke

**Comment by jsingle** [comment]
I feel like we should have one way to run this stuff.  That way we'll be forced to keep this script up to date

**Comment by jsingle** [comment]
Yeah after looking more at the jenkins script and this one I agree it's not worth consolidating them

**Comment by jsingle** [comment]
Looks good to me.  I think it's OK that we're leaving upgrade integration tests out, although it shouldn't be hard to add support

**Pull request merged**